modules/openapi-generator/src/main/resources/cpp-qt5-client/CMakeLists.txt.mustache

```diff
@@ -5,7 +5,11 @@ set(CMAKE_VERBOSE_MAKEFILE ON)
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 set(CMAKE_AUTOMOC ON)
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC -Wall -Wno-unused-variable")
+if (MSVC)
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /W4")
+else (MSVC)
```

Comment:
I'd go with
```
else ()
...
endif ()
```

to make it look more like other programming languges.

Reply:
Got it, thanks